doc/7/core-classes/batch-controller/introduction/index.md

```diff
@@ -0,0 +1,63 @@
+---
+code: false
+type: page
+title: Introduction
+description: BatchController class
+order: 0
+---
+
+# BatchController
+
+<SinceBadge version="auto-version" />
+
+This class is an overload of the document controller and can be switched "as-is" in your code.
+
+It allows to group API action by batches to significantly increase performances.
+
+The following methods will be executed by batch using
+m* actions:
+ - create => mCreate
+ - replace => mReplace
+ - createOrReplace => mCreateOrReplace
+ - update => mUpdate
+ - get => mGet
+ - exists => mGet
+ - delete => mDelete
+
+::: warning
+Standard API errors will not be available.
+Except for the `services.storage.not_found` error.
+:::
+
+By default, the BatchController send a batch of document every 10ms. This can be configured when instantiating the BatchController through the `options.interval` constructor parameter.
```

Comment:
```suggestion
By default, the BatchController sends a batch of documents every 10ms. This can be configured when instantiating the BatchController through the `options.interval` constructor parameter.
```